Makefile

```diff
@@ -0,0 +1,22 @@
+LOCALDIR ?= $(PWD)
+
+REPOS = r4.0-dom0-fc25 r4.0-vm-fc29 r4.0-vm-fc30 r4.0-vm-fc31 r4.0-vm-fc32 r4.1-vm-fc33 r4.0-vm-centos7 r4.0-vm-centos8
+REPOS += r4.1-dom0-fc32 r4.1-vm-fc30 r4.1-vm-fc31 r4.1-vm-fc32 r4.1-vm-fc33 r4.1-vm-centos7 r4.1-vm-centos8
+
+get-sources:
+	@git submodule update --init --recursive
+
+verify-sources:
+	@true
+
+repos: $(addprefix skeleton-,$(REPOS)) $(addprefix comps-,$(REPOS))
+
+comps-%: release=$(word 1,$(subst -, ,$(subst comps-,,$@)))
+comps-%:
+	@cp -r $(LOCALDIR)/meta-packages/comps/comps-dom0.xml $(LOCALDIR)/meta-packages/comps/comps-vm.xml $(LOCALDIR)/$(release)
```

Comment:
This isn't this simple. There are different comps files (especially for dom0) for different qubes versions. Those live in separate branches (in practice even worse - in different repository, because of the recent move).
Perhaps better add some separate sync script instead of using submodule?

Also - `-r`? This is a file, not a directory.

Reply:
Yes mistake.